manifests/server/params.pp

```diff
@@ -85,8 +85,8 @@
   }
 
   if (
-    ($facts['os']['name'] in ['RedHat','CentOS','OracleLinux'] and versioncmp($facts['os']['release']['major'],'7') >= 0) or
-    ($facts['os']['name'] in ['Fedora'] and versioncmp($facts['os']['release']['major'],'22') >= 0)
+    ($facts.dig('os', 'name') in ['RedHat','CentOS','OracleLinux'] and versioncmp($facts.dig('os','release','major'),'7') >= 0) or
+    ($facts.dig('os','name') in ['Fedora'] and versioncmp($facts.dig('os','release','major'),'22') >= 0)
```

Comment:
Should we keep this?  Fedora isn't in the **metadata.json**'s `operatingsystem_support` list

Reply:
This should be removed because `ssh::server`  has `simplib::assert_metadata( $module_name )`